src/skmultiflow/data/random_tree_generator.py

```diff
@@ -99,10 +100,13 @@ class RandomTreeGenerator(Stream):
     -1
     >>> stream.has_more_samples()
     True
-    
+
     """
-    def __init__(self, tree_random_state=None, sample_random_state=None, n_classes=2, n_cat_features=5,
-                 n_num_features=5, n_categories_per_cat_feature=5, max_tree_depth=5, min_leaf_depth=3,
+
+    def __init__(self, tree_random_state=None, sample_random_state=None, n_classes=2,
+                 n_cat_features=5,
+                 n_num_features=5, n_categories_per_cat_feature=5, max_tree_depth=5,
+                 min_leaf_depth=3,
```

Comment:
Are the line breaks in lines 107 and 109 (aesthetically) correct?

Reply:
Done